### ISSUE TITLE
[A11y]Narrator unnecessarily announces table for "Upload Your Package" control

### ISSUE THREAD

**Issue by ryuyu** [NuGet/NuGetGallery#7886]
Copied from https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1069499
Steps to Reproduce:
1. Hit the #URL:https://www.nuget.org/
2. Navigate to "Sign in" control and hit enter.
3. Navigate to "Sign in with Microsoft" control and hit enter.
4. Select “Upload Package” from the Profile Dropdown and hit enter.
5. Turn on narrator navigate to "Upload Your Package" control using "Caps+Arrow" keys.
6. Observe Screen reader announcement.

Actual:
Narrator unnecessarily announces table for "Upload Your Package" control.

Same issue is observed in chrome+NVDA

Image

Expected:
Narrator should announce the proper name of the "Upload Your Package" control. It should not announce it as table.

Notes:
On repro attempt, narrator read out "Upload, Button, Expanded" and did not read out table. Verify that this doesn't repro and we can close it out.

**Comment by ryuyu** [comment]
Completely unable to reproduce. Closed
